### ISSUE TITLE
About the file “synthetic.py” 

### ISSUE THREAD

**Issue by liaoJing56** [bdy9527/FAGCN#6]
Hi! Your work is of great help to my research.
Now I am reproducing Fig. 1(a), but I found that there are only test codes for the low-frequency and high-frequency signals of Fig. 1(a) in  file “synthetic.py”  and no test codes for the FAGCN model. I have tried to write it by myself but the effect is poor.
Could you please help me to reproduce it or supplement the file.
Waiting eagerly for your answer.

**Comment by liaoJing56** [comment]
Specifically, in the section 2, is there no validation set when training the FAGCN model? How to get the best model parameters? How to set some hyperparameters of the model?
If you can provide me with some help, I will be grateful.

**Comment by liaoJing56** [comment]
Thanks to the author, this dataset is under the  file "syn", and we can modify the dataset ourselves to reproduce the results.
